detectable/src/main/java/com/synopsys/integration/detectable/detectables/git/GitUrlParser.java

```diff
@@ -0,0 +1,42 @@
+package com.synopsys.integration.detectable.detectables.git;
+
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class GitUrlParser {
+    // Parses urls such as: https://github.com/blackducksoftware/synopsys-detect
+    public String getRepoName(String remoteUrlString) throws MalformedURLException {
```

Comment:
This method existed before but appears as new because the file moved to a different package and git did not recognize it as such.